smtp/server.go

```diff
@@ -31,6 +31,7 @@ var SendAsSubject = regexp.MustCompile(`(?i)\[sendas:\s*([a-zA-Z0-9\.\-_]+)\]`)
 
 var (
 	ReplyOK               = ReplyLine{250, "OK"}
+	ReplyAuthOk           = ReplyLine{235, "auth success"}
```

Comment:
Can you name this `ReplyAuthOK` (capital K) to match `ReplyOK`?

Reply:
of cause